openwisp_controller/geo/tests/test_api.py

```diff
@@ -172,6 +174,17 @@ def test_geojson_list(self):
             self.assertContains(r, str(location_a.pk))
             self.assertContains(r, str(location_b.pk))
 
+        with self.subTest('Test filtering using organization slug'):
+            self.client.login(username='admin', password='tester')
+            response = self.client.get(
+                reverse(url), data={'organization_slug': org_a.slug}
```

Comment:
>.https://github.com/openwisp/openwisp-firmware-upgrader/blob/8783ea12f9a1963e2f8c5d49abe8bcaadaa13a2d/openwisp_firmware_upgrader/api/views.py#L48-L56

I think we should update that logic in favour of django-filters.

Reply:
That one is not really a filter and is going to be replaced by https://github.com/openwisp/openwisp-firmware-upgrader/pull/150, which I haven't merged yet because the number of queries increase and I want to be 100% sure why this is happening before merging.